README.rst

```diff
@@ -170,9 +177,9 @@ Roll your own context_parser
     logger = pypyr.log.logger.get_logger(__name__)
 
 
-    def get_parsed_context(context):
+    def get_parsed_context(context_string):
         """This is the signature for a context parser. Input context is the string received from pypyr --context 'value here'"""
-        assert context, ("pipeline must be invoked with --context set.")
+        assert context_string, ("pipeline must be invoked with --context set.")
```

Comment:
Can't quite tell from this, but it looks like context_string is actually the dictionary key.  If this is the case then it would be better to name it a context_key rather than a context_string.  Makes it more obvious what the intended usage is.